Automattic/wp-calypso
# Fixes a styling issue in the Blaze Dashboard app

Fixes some styling issues in the Blaze Dashboard app. 

![Screenshot 2023-06-29 at 10 49 17](https://github.com/Automattic/wp-calypso/assets/1258162/788b9d19-6684-4e28-8c1e-415141206bb7)

## Proposed Changes

* Changed the way we solved a conflicting stylesheet to be more generic. The previous version only worked correctly in self-hosted sites (not in atomic).
* Fixed a visibility problem with the Create campaign widget popovers
* Fixed a visibility problem with the Create campaign widget date pickers

## Testing Instructions
You will need a Jetpack local environment running to test this. Also, you will need to be able to execute a code snippet that will open a gate for the Blaze Dashboard. To do so, follow these instructions:
* Install the Plugin Code Snippets
* Create a new snippet with the following content: `add_filter( 'jetpack_blaze_dashboard_enable', '__return_true' );`

---

* Checkout this branch, then open a terminal and navigate to wp-calypso/apps/blaze-dashboard
* Execute `BLAZE_DASHBOARD_PACKAGE_PATH=/path/to/jetpack/projects/packages/blaze yarn dev`
You will need to change the path to your local Jetpack installation.
* In another terminal tab, go to the Jetpack repo directory and execute: `pnpm jetpack build packages/blaze`
* Now open your Jurassic tunnel and enter the wp-admin page for your local site
* Click on Tools->Advertising
* Click on the Promote button next to the Jetpack Blaze header
* Verify that the search input in the **Select post to promote** page is displayed correctly.
* Shrink your screen into a mobile view
* Click on the three-dot menu for one of the rows
* Verify that the popover is displayed correctly
* Click on the Promote button
* Verify that all the inputs of that page are displayed correctly
* Click on the calendar icon in the **Start date** section
* Verify that the Date picker popover is displayed correctly

## Pre-merge Checklist
